core/src/main/scala/org/broadinstitute/dsde/rawls/util/WorkspaceSupport.scala

```diff
@@ -130,28 +121,36 @@ trait WorkspaceSupport {
   def getV2WorkspaceContextAndPermissions(
     workspaceName: WorkspaceName,
     requiredAction: SamResourceAction,
-    attributeSpecs: Option[WorkspaceAttributeSpecs] = None
+    attributeSpecs: Option[WorkspaceAttributeSpecs] = None,
+    ignoreLock: Boolean = false
```

Comment:
Is this ever called with `ignoreLock` = True? I'm not seeing where that is, and it would be great to simplify and remove the argument if possible.

Reply:
Yes: specifically, when locking and unlocking a workspace.